### PR TITLE
Update the Docusaurus Social Card with Obscuro Banner

### PR DESCRIPTION
**Pull request by 0xjba**

### CODE REVIEW

**Review by StefanIliev545 — APPROVED** [review]
LGTM

File: docs/tools-infrastructure/hosted-gateway.md [hunk 1]
```diff
@@ -4,3 +4,46 @@ sidebar_position: 3
 
 # Hosted Gateway
 
+The Obscuro Gateway is a critical application that facilitates communication between an Obscuro node and various tools that require a connection to it, such as MetaMask. Due to the encryption of data within an Obscuro node, direct communication is not feasible.
+
+The program conforms to the Ethereum JSON-RPC specification ([Ethereum JSON-RPC Specification](https://playground.open-rpc.org/?schemaUrl=https://raw.githubusercontent.com/ethereum/eth1.0-apis/assembled-spec/openrpc.json)) and also supports additional APIs to ensure compatibility with popular tools like MetaMask.
+
+You have the flexibility to host the Obscuro Gateway yourself or use one of the hosted gateways if you choose to join Obscuro. If security is your primary concern, you can run and use the program independently. The diagram below illustrates different usage scenarios, with Bob and Charlie using the hosted version and Alice managing it independently.
```
Comment (tudor-malene) [inline docs/tools-infrastructure/hosted-gateway.md:11]: I would replace 'if security is your primary concern" with "you have the option to."

File: docs/introduction/developer-quickstart.md [hunk 2]
```diff
@@ -6,22 +6,24 @@ sidebar_position: 4
 
 Obscuro executes bytecode in the EVM no different than Ethereum (or Arbitrum), so as a developer you can use all your existing code and tooling.
```
Comment (CaisManai) [inline docs/introduction/developer-quickstart.md:7]: I would change this line to:

Obscuro performs bytecode execution in the EVM identically to Ethereum, allowing developers to leverage their existing codebase and tools.

File: docs/introduction/developer-quickstart.md [hunk 3]
```diff
@@ -6,22 +6,24 @@ sidebar_position: 4
...
 
-The difference from Ethereum (and virtually all chains) is that private variables and the internal state of the contract while 
-executing are hidden from everyone (which includes node operators and the sequencer).
+The difference from Ethereum (and virtually all chains) is that private variables and the internal state of the contract while executing are hidden from everyone (which includes node operators and the sequencer).
```
Comment (CaisManai) [inline docs/introduction/developer-quickstart.md:9]: Rewrite as below to make it sound more positive. 

The main difference and advantage of Obscuro is that on Obscuro, during execution, private variables and the internal state of the contract are hidden from everyone, including node operators and the sequencer.

File: docs/introduction/developer-quickstart.md [hunk 4]
```diff
@@ -6,22 +6,24 @@ sidebar_position: 4
...
 :::
 
-The calls to [getStorageAt](https://docs.alchemy.com/reference/eth-getstorageat) are disabled, so all data access
-will be performed through view functions which are under the control of the smart contract developer.
-Public variables are accessible to everyone because Solidity automatically generates a getter function for them.
+The calls to [getStorageAt](https://docs.alchemy.com/reference/eth-getstorageat) are disabled, so all data access will be performed through view functions which are under the control of the smart contract developer. Public variables are accessible to everyone because Solidity automatically generates a getter function for them.
 
-We'll illustrate how it works by creating a simple data storage example.
-In this dApp, users can store a number and retrieve it later.
+We'll illustrate how it works by creating a simple data storage example. In this dApp, users can store a number and retrieve it later.
```
Comment (CaisManai) [inline docs/introduction/developer-quickstart.md:17]: illustrate how **this** works

File: docs/introduction/developer-quickstart.md [hunk 5]
```diff
@@ -91,22 +90,17 @@ contract StorageExample {
...
-there is a signature of the ``tx.origin`` address. 
-
+In Obscuro, the platform ensures that calls to view functions are authenticated. Which means that behind the scenes, there is a signature of the `tx.origin` address.
 
 ## Step 4: Emitting Events
 
-Events in Ethereum are crucial for UIs to react to smart contract state changes.
-In this step we'll emit an event when a user stores a value, and also, to show how popular our contract is,
-we'll emit an event when certain milestones are reached.
+Events in Ethereum are crucial for UIs to react to smart contract state changes. In this step, we'll emit an event when a user stores a value, and also, to show how popular our contract is, we'll emit an event when certain milestones are reached.
```
Comment (CaisManai) [inline docs/introduction/developer-quickstart.md:103]: Events in Ethereum are crucial for UIs to react to smart contract state changes. In this step, we'll emit an event when a user stores a value. We'll also gauge the popularity of our contract by emitting an event when certain milestones are reached.

**Review by CaisManai — CHANGES_REQUESTED** [review]

File: docs/introduction/developer-quickstart.md (hunk 2 above)
Comment (0xjba) [reply docs/introduction/developer-quickstart.md:7]: Fixed

File: docs/introduction/developer-quickstart.md (hunk 3 above)
Comment (0xjba) [reply docs/introduction/developer-quickstart.md:9]: Fixed

File: docs/introduction/developer-quickstart.md (hunk 5 above)
Comment (0xjba) [reply docs/introduction/developer-quickstart.md:103]: Fixed

File: docs/introduction/developer-quickstart.md (hunk 4 above)
Comment (0xjba) [reply docs/introduction/developer-quickstart.md]: Fixed

**Review by 0xjba — COMMENTED** [review]
Included all requested changes

**Review by tudor-malene — APPROVED** [review]
lgtm.
(With a doubt that this will render properly, so it might require a few tweaks)

**Review by CaisManai — APPROVED** [review]

**Pull request merged**